Sources/BeaconChain/StateTransition.swift

```diff
@@ -297,7 +340,7 @@ extension StateTransition {
 extension StateTransition {
 
     static func processEpoch(state: inout BeaconState) {
-        assert(state.slot + 1 % EPOCH_LENGTH == 0) // @todo not sure if this should be here
+        assert(state.slot + 1 % SLOTS_PER_EPOCH == 0) // @todo not sure if this should be here
```

Comment:
You should only call this function if that condition holds. Seems a little redundant to assert it here but not necessarily incorrect